test/cqlpy/test_describe.py

```diff
@@ -2746,6 +2747,7 @@ def test_desc_restore(cql):
                             WITH comment='some other comment'""")
 
         cql.execute(f"CREATE INDEX myindex ON {ks}.some_other_table (c1)")
+        cql.execute(f"CREATE INDEX custom_index ON {ks}.vector_table (v) USING 'vector_index'")
```

Comment:
This vector index made this test scylla_only, but I'm guessing it already was because it uses service levels.

In general, I would prefer not to test new features in old tests - we should have new tests for the vector index that aren't mixed with service levels tests and other stuff - but this test is already so weird that I won't insist.